packages/commerce-sdk-react/src/provider.tsx

```diff
@@ -77,7 +77,9 @@ const CommerceApiProvider = (props: CommerceApiProviderProps): ReactElement => {
             clientId,
             organizationId,
             shortCode,
-            siteId
+            siteId,
+            locale,
+            currency
```

Comment:
Should these also be added to `ApiClientConfigParams`, since that's what we're doing, now?

Reply:
They are already in the type

Reply:
Doesn't look like that's the case:
https://github.com/SalesforceCommerceCloud/pwa-kit/blob/7db73df25b72df3572d1c857e5f47012395ba8db/packages/commerce-sdk-react/src/hooks/types.ts#L70-L75

The `ApiClientConfigParams` should include all the parameters we pass to it, otherwise we'll see a type error when we try to access the parameters.
```ts
useCommerceApi().shopperBaskets.clientConfig.parameters.currency // Error :(
```

Reply:
ah I see, gotcha, I will add them in. I looked at the ProviderProps yesterday 😅